cyw43/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 #![no_std]
 #![no_main]
-#![feature(async_fn_in_trait, type_alias_impl_trait, concat_bytes)]
+#![feature(async_fn_in_trait, type_alias_impl_trait, concat_bytes, slice_as_chunks)]
```

Comment:
We shouldn't add more nightly features, the goal is for Embassy to work on stable when AFIT and TAIT are stable.

Reply:
Noted, I'll do it without. 👍